app/src/main/java/com/xa/xpensauditor/MonthWiseVis.java

```diff
@@ -121,14 +125,13 @@ protected void onCreate(Bundle savedInstanceState) {
 
             if (f) {
                 System.out.println("Day_Wise   "+m2 +"    "+y2);
-                for(int ii=m1;ii<=m2;ii++){
+                for( ii=m1;ii<=m2;ii++){
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '=' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)